pyswarms/base/base_single.py

```diff
@@ -82,6 +82,14 @@ def assertions(self):
                 raise ValueError('Make sure that velocity_clamp is in the '
                                  'form (min, max)')
 
+        # Check setting of guess
```

Comment:
Just wondering about the use of the word `guess`. We are setting for the initial position of the particles, right? Instead of having it be defined by our bounds. I'd suggest we go for `init_pos` or something that is more explicit. We can update the documentation to mention the word "guess" instead.